tests/stdlib/thashes.nim

```diff
@@ -1,3 +1,7 @@
+discard """
+targets: '''c c++ js'''
```

Comment:
```suggestion
  targets: '''c c++ js'''
```

Reply:
@xflywind also, all targets: c++ have been converted to cpp except this => i'm fixing this in https://github.com/nim-lang/Nim/pull/16343